src/main/java/se/iths/librarysystem/entity/BookEntity.java

```diff
@@ -17,7 +17,7 @@
     @NotBlank(message = "Edition is a required field")
     private String edition;
     @NotNull(message = "Print-date is a required field")
-    private Date printed;
+    private String printed;
```

Comment:
Do we want the field `printed` to be a String instead of a Date object? It is possible to submit a JSON request body with the date in string format (YYYY-MM-DD) and Spring Boot will convert it into a Date object for us.